### DISCUSSION TITLE
Welcome to SensESP Discussions!

### DISCUSSION THREAD

**Discussion by mairas** [SignalK/SensESP#540] [Announcements]
## 👋 Welcome, fellow SensESP user/developer/person/entity!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by euxneri** [comment]
I have built according to your site instructions the one- wire temp device 

( i am closed at home with the good company of a corona virus - not beer😡-)

Everything went smoothly but signalk does not receives 
The device was accepted by the server ( I authorized) and i can see among networks “ configure temperature” ( it asks a key PSK ) that i have not . 

Please any suggestion ? The psk key?

Thank you
Eugenio 

ps : the password for the network “ configure temperature “ I found to be “thisisfine” . 
i checked avahi is ok) 
 However no way to see the data connection on signal k 
( debian 11 64 bits opencpn v3 sk1.42)

**Comment by euxneri** [comment]
![C8D539A6-1CB3-4868-88FD-435E3BEABE7B](https://user-images.githubusercontent.com/12800659/165045970-b431b60f-f371-45fd-8234-a504171ddd8b.jpeg)
